po/ja.po

```diff
@@ -9206,192 +9319,208 @@ msgid ""
 
 #: src/modules/solution.md
 msgid "// ANCHOR_END: Label-width\n"
-msgstr ""
+msgstr "// ANCHOR_END: Label-width\n"
 
 #: src/modules/solution.md
 msgid "// ANCHOR: Label-draw_into\n"
-msgstr ""
+msgstr "// ANCHOR: Label-draw_into\n"
 
 #: src/modules/solution.md
 msgid "// ANCHOR_END: Label-draw_into\n"
-msgstr ""
+msgstr "// ANCHOR_END: Label-draw_into\n"
 
 #: src/modules/solution.md
 msgid "// ---- src/widgets/button.rs ----\n"
-msgstr ""
+msgstr "// ---- src/widgets/button.rs ----\n"
 
 #: src/modules/solution.md
 msgid "// ANCHOR_END: Button-width\n"
-msgstr ""
+msgstr "// ANCHOR_END: Button-width\n"
 
 #: src/modules/solution.md
 msgid "// ANCHOR: Button-draw_into\n"
-msgstr ""
+msgstr "// ANCHOR: Button-draw_into\n"
 
 #: src/modules/solution.md
 msgid "// ANCHOR_END: Button-draw_into\n"
-msgstr ""
+msgstr "// ANCHOR_END: Button-draw_into\n"
 
 #: src/modules/solution.md
 msgid "// ---- src/widgets/window.rs ----\n"
-msgstr ""
+msgstr "// ---- src/widgets/window.rs ----\n"
 
 #: src/modules/solution.md
 msgid ""
 "// ANCHOR_END: Window-width\n"
 "        // Add 4 paddings for borders\n"
 msgstr ""
+"// ANCHOR_END: Window-width\n"
+"        // 枠線に 4 つのパディングを追加します。\n"
 
 #: src/modules/solution.md
 msgid "// ANCHOR: Window-draw_into\n"
-msgstr ""
+msgstr "// ANCHOR: Window-draw_into\n"
 
 #: src/modules/solution.md
 msgid "// ANCHOR_END: Window-draw_into\n"
-msgstr ""
+msgstr "// ANCHOR_END: Window-draw_into\n"
 
 #: src/modules/solution.md
 msgid ""
 "// TODO: after learning about error handling, you can change\n"
 "        // draw_into to return Result<(), std::fmt::Error>. Then use\n"
 "        // the ?-operator here instead of .unwrap().\n"
 msgstr ""
+"// TODO: エラー処理について学習した後で、\n"
+"        // Result<(), std::fmt::Error> を返すように draw_into を変更できま"
+"す。次に、ここで\n"
+"        // .unwrap() の代わりに ? 演算子を使用します。\n"
 
 #: src/modules/solution.md
 msgid "// ---- src/main.rs ----\n"
-msgstr ""
+msgstr "// ---- src/main.rs ----\n"
 
 #: src/testing.md
 msgid "[Test Modules](./testing/unit-tests.md) (5 minutes)"
-msgstr ""
+msgstr "[テスト] モジュール(./testing/unit-tests.md)（5 分）"
 
 #: src/testing.md
 msgid "[Other Types of Tests](./testing/other.md) (10 minutes)"
-msgstr ""
+msgstr "[その他の種類のテスト](./testing/other.md)（10 分）"
 
 #: src/testing.md
 msgid "[Useful Crates](./testing/useful-crates.md) (3 minutes)"
-msgstr ""
+msgstr "[便利なクレート](./testing/useful-crates.md)（3 分）"
 
 #: src/testing.md
 msgid "[GoogleTest](./testing/googletest.md) (5 minutes)"
-msgstr ""
+msgstr "[GoogleTest](./testing/googletest.md)（5 分）"
 
 #: src/testing.md
 msgid "[Mocking](./testing/mocking.md) (5 minutes)"
-msgstr ""
+msgstr "[モック](./testing/mocking.md)（5 分）"
 
 #: src/testing.md
 msgid "[Compiler Lints and Clippy](./testing/lints.md) (5 minutes)"
-msgstr ""
+msgstr "[コンパイラの リント と Clippy](./testing/lints.md)（5 分）"
 
 #: src/testing.md
 msgid "[Exercise: Luhn Algorithm](./testing/exercise.md) (30 minutes)"
-msgstr ""
+msgstr "[演習: Luhn アルゴリズム](./testing/exercise.md)（30 分）"
```

Comment:
It seems this is not used at all. The one in line 9893 is used. Maybe a bug in mdbook or somewhere?